src/pages/workspaces/workspace/Data.js

```diff
@@ -194,6 +202,54 @@ class WorkspaceData extends Component {
     ])
   }
 
+  renderDownloadButton() {
+    const { namespace, name } = this.props
+    const { selectedDataType, orchestrationRoot } = this.state
+
+    return form({
+      style: { display: 'inline' },
+      target: '_blank', method: 'POST',
+      action: `${orchestrationRoot}/cookie-authed/workspaces/${namespace}/${name}/entities/${selectedDataType}/tsv`
+    }, [
+      input({ type: 'hidden', name: 'FCtoken', value: auth.getAuthToken() }),
+      /*
+       * TODO: once column selection is implemented, add another hidden input with name: 'attributeNames' and
+       * value: comma-separated list of attribute names to support downloading only the selected columns
+       */
+      buttonPrimary({ type: 'submit' }, [
```

Comment:
Should this be disabled (with a message?) while the Orch URL loads just in case it's slow?